tests/FakeItEasy.IntegrationTests/ConcurrentCallTests.cs

```diff
@@ -125,11 +125,9 @@ public override void MutateRules(IFoo fake)
             }
         }
 
-        private sealed class ClearConfiguration : RuleMutatingAction
+        private sealed class ResetFake : RuleMutatingAction

```

Comment:
I could just remove this completely if you prefer.

Reply:
No strong opinion. Easy to remove later if we feel like it. Let's keep it.